program-runtime/src/loaded_programs.rs

```diff
@@ -862,7 +883,7 @@ impl<FG: ForkGraph> LoadedPrograms<FG> {
                             if let LoadedProgramType::Unloaded(_environment) = &entry.program {
                                 break;
                             }
-
+                            entry.update_access_slot(loaded_programs_for_tx_batch.slot);
```

Comment:
Why only update it in this case here and not below where we also increment the usage counter?

Reply:
The test uses it too `test_usage_counter_decay`.

Reply:
Since we are sticking with Atomics, it'll be good to have a wrapper for test and the code.